cmd/cmd.go

```diff
@@ -85,6 +85,22 @@ func runScan(cmd *cobra.Command, args []string) {
 		logger.Fatal("invalid leak-exit-code: %v", err)
 	}
 
+	gitleaksConfig, err := cmd.Flags().GetString("gitleaks-config")
+	if err != nil {
+		logger.Fatal("invalid gitleaks-config: error=%q", err.Error())
+	}
+
+	// Providing a gitleaks-config via command line arguments takes
+	// precedence over gitleaks config set in the leaktk config file
+	if len(gitleaksConfig) != 0 {
+		cfg.Scanner.Patterns.Gitleaks.ConfigPath = gitleaksConfig
+		logger.Debug("using provided gitleaks config: path=%s", gitleaksConfig)
```

Comment:
Should this log be near here instead? https://github.com/leaktk/leaktk/blob/cc536e70a414d5c72dffc52be0ff78b840612ac5/pkg/scanner/patterns.go#L139

Reply:
Tentatively moved the log to there.